### PR TITLE
添加eslint&自动化构建

### PR DESCRIPTION
**Pull request by wenmine**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] CI related changes
- [ ] Documentation content changes
- [] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
添加eslint&自动化构建，每次push，创建PR时都会触发此流程，保障基础门禁

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/vue-generator/test/testcases/full/case1_normal/output/views/FormTable.vue [hunk 1]
```diff
@@ -199,7 +199,7 @@ const state = vue.reactive({
         ]
       },
       slots: {
-        default: ({ row }, h) => (
+        default: ({ row }) => (
```
Comment (chilingling) [inline packages/vue-generator/test/testcases/full/case1_normal/output/views/FormTable.vue:202]: 这里是 output 的数据，应该要 ignore 的，我在 #52  已经做了 ignore 了，可以撤回一下

File: .github/workflows/push-check.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      uses: tj-actions/changed-files@v40
+      with:
+        files: |
+              **.js
+              **.vue
+              **.jsx
+    - name: Run ESLint
+      run: npx eslint ${{steps.get_changed_files.outputs.all_changed_files}}
+    - name: Run Build
+      run: pnpm run build:plugin
```
Comment (chilingling) [inline .github/workflows/push-check.yml]: 这里还要构建一下 alpha ： `pnpm build:alpha`

File: .github/workflows/push-check.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+name: CI
```
Comment (chilingling) [inline .github/workflows/push-check.yml]: 名字可以语义化一下

**Review by chilingling — APPROVED** [review]
Nice

**Review by gargameljyh — APPROVED** [review]

**Review by gargameljyh — APPROVED** [review]

**Review by gargameljyh — APPROVED** [review]

**Pull request merged**